cristin-project/src/main/java/no/unit/nva/cristin/projects/common/ParameterKeyProject.java

```diff
@@ -174,6 +203,10 @@ private static boolean ignorePathKeys(ParameterKeyProject f) {
         return f.ordinal() > IGNORE_PATH_PARAMETER_INDEX;
     }
 
+    private static boolean ignoreFacetKeys(ParameterKeyProject f) {
```

Comment:
`f` maybe means something…

Reply:
copy paste from existing code. should i name it something else or just follow the pattern?